src/lib/common/sol-log-internal.h

```diff
@@ -36,7 +36,7 @@
 
 #ifdef SOL_LOG_ENABLED
 #define SOL_LOG_INTERNAL_DECLARE(_var, _name)    \
-    struct sol_log_domain _var = {               \
+    SOL_ATTR_UNUSED struct sol_log_domain _var = { \
```

Comment:
Yeah, by only removing the change on the 2nd file we'll have to depend on the order of inclusion in all node files (node's generated header vs. that sol-log-internal.h), which is bad.